frontend/public/components/custom-resource-definition.tsx

```diff
@@ -231,7 +231,7 @@ const Details: React.FC<{ obj: CustomResourceDefinitionKind }> = ({ obj: crd })
                   <Established crd={crd} />
                 </dd>
                 <DetailsItem label="Group" obj={crd} path="spec.group" />
-                <DetailsItem label="Version" obj={crd} path="spec.version" />
+                <DetailsItem label="Version" obj={crd} path="spec.versions[0].name" />
```

Comment:
Should this be updated to handle multiple versions better?

Reply:
Or maybe the column heading should be "Latest Version" if we just show the first item in the versions array.

Reply:
Updated col head to `Latest Version`